src/index.js

```diff
@@ -15,6 +15,12 @@ class IframeResizer extends React.Component {
     this.updateIframe(this.props);
     this.resizeIframe(this.props);
   }
+  componentWillUnmount() {
+    // React will remove the iframe, however we need to manually
+    // call iframe-resizer to stop its listeners
+    iframeResizer = this.refs.frame.iFrameResizer
```

Comment:
I am getting an error:

`iframeResizer is not defined`

This line needs to be changed to:

`let iframeResizer = this.refs.frame.iFrameResizer`